apps/next/src/pages/LockScreen/components/Unlock.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box } from '@avalabs/k2-alpine';
+import { Stack, Button } from '@avalabs/k2-alpine';
+import { PasswordField } from '@/components/StandaloneField';
+
+type Props = {
+  onUnlock: (password: string) => Promise<true>;
+  onForgotPasswordClick: () => void;
+};
+
+export const Unlock: React.FC<Props> = ({
+  onUnlock,
+  onForgotPasswordClick,
+}) => {
+  const { t } = useTranslation();
+  const [password, setPassword] = useState<string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<string>('');
```

Comment:
No need for `<type>` I guess, TS should infer properly.

```suggestion
  const [password, setPassword] = useState('');
  const [isLoading, setIsLoading] = useState(false);
  const [error, setError] = useState('');
```